src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.cs

```diff
@@ -156,24 +157,15 @@ private static bool AnyImplementedImplicitly(State state)
             return false;
         }
 
-        private static IList<ISymbol> GetDelegatableMembers(State state)
+        private static ImmutableArray<ISymbol> GetDelegatableMembers(State state, CancellationToken cancellationToken)
         {
-            var fields =
```

Comment:
moved this code into helper type.  That way we could add support for primary constructor parameters, and have teh same logic for 'implement interface' and 'implement abstract class'.